log4j-1.2-api/src/test/java/org/apache/log4j/CategoryTest.java

```diff
@@ -78,23 +78,23 @@ public void testForcedLog() {
         category.info("Hello, World");
         final List<LogEvent> list = appender.getEvents();
         int events = list.size();
-        assertTrue("Number of events should be 1, was " + events, events == 1);
+        assertThat(events).describedAs("Number of events should be 1, was " + events).isEqualTo(1);
         LogEvent event = list.get(0);
         Message msg = event.getMessage();
-        assertNotNull("No message", msg);
-        assertTrue("Incorrect Message type", msg instanceof ObjectMessage);
+        assertThat(msg).describedAs("No message").isNotNull();
+        assertThat(msg instanceof ObjectMessage).describedAs("Incorrect Message type").isTrue();
```

Comment:
I thought I'd written a transformation for these, perhaps not!
Should end up with:
```suggestion
        assertThat(msg).describedAs("Incorrect Message type").isInstanceof(ObjectMessage.class);
```